### ISSUE TITLE
Make folder settings a page

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#4088]
The modal dialog for the folder properties is a design alien.

**Goal:** It should be adapted to the general program design and be presented as a separate page:

![Example folder settings page](https://user-images.githubusercontent.com/3040657/98353257-5bbb2b00-201f-11eb-8321-1ecd8de2c36b.png)

**Comment by matthias-ronge** [comment]
Note: Pending changes like #3371, #3659 will require even more display space in the folder dialog.

**Comment by solth** [comment]
I disagree. The modal dialog allows us to edit the folder properties during project configuration. If we add a button to navigate from "projectEdit" to a new page "folderEdit", we have the same problem that currently exists when editing tasks of an existing process: when you navigate from "processEdit" to "taskEdit", you have a "Save" button there to persist changes made to the task, but then return to the "processEdit" page, where it isn't clear if the task changes are really already saved or only if "Save" is clicked again, this time on the "processEdit" page. 

We would face the same ambiguity when we introduce a new "folderEdit" page that can be reached from "projectEdit".

**Comment by matthias-ronge** [comment]
That is an argument. I know the situation from other web applications in which you actually have to save twice. Perhaps instead of "Save" it would be better to label the button "OK" or "Apply" in that case. However, we can stick to the modal dialog if you want. But there is the question, if there is _more_ information that has to be provided, is it possible to make several tabs in dialogs, something like Windows:

![Screenshot](https://user-images.githubusercontent.com/3040657/124115545-38416680-da6e-11eb-97af-fc9d8b7c2afe.png)
